docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,9 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "hostExpiryEnabled": true,
   "mdmWindowsEnabled": false,
   "liveQueryDisabled": false,
+  "aiFeaturesDisabled": false,
+  "maintenanceWindowsEnabled": true,
+  "fleetDesktopEnabled": true,
```

Comment:
##### Dev note
Whether at least 1 host has a value for `fleet_desktop_version`

Reply:
> host has a value for fleet_desktop_version

Measuring based on version makes sense to me.

But I think we should capture the number of hosts w/ Fleet Desktop enabled. This way, we can measure rollout as a heuristic to measure usage. For example, we can decide (and change our minds) on whether we say a customer is using Fleet Desktop if they have it rolled out to 50% of their hosts.

What do you think?

Reply:
cc @rachaelshaw ^

Reply:
@noahtalerman That makes total sense to me, a count is definitely more useful than a boolean here. That way it doesn't look like an org is using Fleet Desktop when they may just be trying it out on a few hosts & haven't rolled it out to everyone else yet. Also, we can compare that count to the number of workstations, rather than having an arbitrary % where we consider it to be enabled.

I'll update this to `numHostsFleetDesktopEnabled`.